wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+            patterns: [
+                "*.fq.gz",
+                "*.fastq.gz",
+            ],
+        }
+        results_name: {
+            description: "Experiment results custom name",
+            group: "analysis_parameter",
+            help: "Input preferred analysis results name (recommended if multiple FASTQs are provided).",
+            example: "AllMerge_mapped",
+        }
+        run_motifs: {
+            description: "Perform Motif Analysis",
+            group: "analysis_parameter",
+            help: "Setting this means Motif Discovery and Enrichment analysis will be performed.",
+            example: true,
+        }
+    }
+
+    input {
+        # group: reference_genome
+        File reference
+        File? spikein_reference
+        File? blacklist
+        File gtf
+        Array[File]? bowtie_index
+        Array[File]? spikein_bowtie_index
+        Array[File]? motif_databases
+
+        # group: input_genomic_data
+        Array[String]? sample_sraid
+        Array[File]? sample_fastq
+
+        # group: analysis_parameter
+        String? results_name
+        Boolean run_motifs = true
+
+    }
+
+    String pipeline_ver = "v2.0.0"
+
+    ### ---------------------------------------- ###
+    ### ------------ S E C T I O N 1 ----------- ###
+    ### ------ Pre-process Analysis Files ------ ###
+    ### ---------------------------------------- ###
+
+    # Process SRRs
+    if (defined(sample_sraid)) {
+        # Download sample file(s) from SRA database
+        # outputs:
+        #    fastqdump.fastqfile : downloaded sample files in fastq.gz format
+        Array[String] string_sra = [
+            1,
+        ]  #buffer to allow for sra_id optionality
+        Array[String] s_sraid = select_first([
+            sample_sraid,
+            string_sra,
+        ])
+        scatter (eachsra in s_sraid) {
+            call sra.fastqdump { input:
+                sra_id = eachsra,
+                cloud = false,
+            }
+        }  # end scatter each sra
+
+        Array[File] sample_srafile = flatten(fastqdump.fastqfile)
+    }  # end if sample_sraid
+
+    # Generating INDEX files
+    #1. Bowtie INDEX files if not provided
+    if (!defined(bowtie_index)) {
+        # create bowtie index when not provided
+        call bowtie.index as bowtie_idx { input: reference = reference }
+    }
+    #2. Make sure indexes are six else build indexes
+    if (defined(bowtie_index)) {
+        # check total number of bowtie indexes provided
+        Array[String] string_bowtie_index = [
+            1,
+        ]  #buffer to allow for bowtie_index optionality
+        Array[File] int_bowtie_index = select_first([
+            bowtie_index,
+            string_bowtie_index,
+        ])
+        if (length(int_bowtie_index) != 6) {
+            # create bowtie index if 6 index files aren't provided
+            call bowtie.index as bowtie_idx_2 { input: reference = reference }
```

Comment:
Should the formatter handle `reference = reference` and just replace it with `reference`?  Maybe not in this specific case since the document is v1.0

Reply:
I think that's a good idea, which is complicated by how trivia is handled. I don't have an immediate solution for something like this:
```
{ input: reference =  # I'm an interrupting comment
    reference
}
```
I'm fairly confident we don't want to delete any comments. And since this particular comment would be "owned" by the `=` sign, I _think_ we have to include the `=` sign in the output.

Reply:
That's certainly complicated. I think the "right" output becomes
```
{ input: reference  # I'm an interrupting comment }
```
It probably has to be a special case where the comment is re-attached to the LHS of the assignment.